include/my_bit.h

```diff
@@ -38,6 +38,14 @@ static inline uint my_bit_log2(ulong value)
 }
 
 
+static inline uint my_bit_size_t_log2(size_t value)
```

Comment:
you could just modify the function above my_bit_log2() , so it accept size_t . it is always larger or equal type to ulong,
i.e truncations from ulong to size_t do not happen.

Reply:
I couldn't see this in the C11 standard, but happy to accept that is true and put in a compile assertion just to be sure. Thanks for the review.